apps/challenges/permissions.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import permissions
+from .models import Challenge
```

Comment:
@RishabhJain2018 there should be a gap between line 1 and line 2. Also, there should be no gap between the lines `from .models import Challenge` and `from .serializers import ChallengeSerializer`